src/error/locatedError.js

```diff
@@ -17,7 +18,7 @@ import { GraphQLError } from './GraphQLError';
  * document responsible for the original Error.
  */
 export function locatedError(
```

Comment:
I believe the right change here is to add a third `path` argument to `locatedError`